### DISCUSSION TITLE
app like xhunter ?

### DISCUSSION THREAD

**Discussion by hunte1r** [anirudhmalik/xhunter#51] [General]
Do you know an Android app that competes with xhunter or something similar? Put a link below to share the benefit

**Comment by anirudhmalik** [comment]
Closing in favor of v2. See [v2.0-demo](https://github.com/anirudhmalik/xhunter/releases/tag/v2.0-demo) and the repo README. New bugs: open a fresh issue with repro steps. Thanks.
